src/components/Book.js

```diff
@@ -0,0 +1,42 @@
+// Individual Book component
+// import React from 'react';
```

Comment:
- [ ] Leaving unused code as a comment in your codebase is not a best practice, I'll suggest you delete this. :+1: :+1: